core/src/main/java/tech/tablesaw/api/BooleanColumn.java

```diff
@@ -447,15 +448,22 @@ public BooleanColumn set(int i, boolean b) {
     return this;
   }
 
-  private void set(int i, byte b) {
+  private BooleanColumn set(int i, byte b) {
     data.set(i, b);
+    return this;
   }
 
   @Override
   public BooleanColumn set(int i, Boolean val) {
     return set(i, val.booleanValue());
   }
 
+  @Override
+  public Column<Boolean> setValue(int row, String newValue, AbstractColumnParser<?> parser) {
```

Comment:
Can we just call this `set` instead of `setValue`?

Reply:
done.